mintapi/api.py

```diff
@@ -80,7 +80,11 @@ def get_email_code(
             DeprecationWarning,
         )
     code = None
-    imap_client = imaplib.IMAP4_SSL(imap_server)
+    try:
+        imap_client = imaplib.IMAP4_SSL(imap_server)
+    except imaplib.IMAP4.error:
+        logger.error("ERROR: Unable to establish IMAP Client")
+        return ""
```

Comment:
@jaron-l I do agree, but I think that should be done as a follow-up instead of as a part of this particular PR.  I simply added this here as it was an opportunity for unexpected behavior as it stood with using keyring for the IMAP password.

I can do the follow-up work on raising true exceptions for these IMAP errors.

Reply:
Sounds good to me. I went ahead and made #362 to track this.